tgstation.dme

```diff
@@ -2196,8 +2196,8 @@
 #include "code\modules\library\skill_learning\skill_station.dm"
 #include "code\modules\library\skill_learning\skillchip.dm"
 #include "code\modules\library\skill_learning\job_skillchips\_job.dm"
-#include "code\modules\library\skill_learning\job_skillchips\research_director.dm"
 #include "code\modules\library\skill_learning\job_skillchips\psychologist.dm"
+#include "code\modules\library\skill_learning\job_skillchips\research_director.dm"
```

Comment:
This was automatic, not done by me. Im assuming its because of not updating with master or whatnot. Shouldn't really change much.

Reply:
It should be good, it's just changing it to be in alphabetical order, I wouldn't worry about that.